### PR TITLE
[WIP] Security Reports

### PR DESCRIPTION
**Pull request by aabdulwahed**
## Description
- Generate Securify, Mythril and Surya reports.
- Travis should run securify and mythril only in case of merging to develop
- The generated reports will be committed to another private [repo](https://github.com/oceanprotocol/keeper-contracts-security-reports) for analysis and review

## Is this PR related with an open issue?

Related to Issue [#206](https://github.com/oceanprotocol/keeper-contracts/issues/206), [#97](https://github.com/oceanprotocol/keeper-contracts/issues/97), [252](https://github.com/oceanprotocol/keeper-contracts/issues/252)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
 
## Checklist:

- [ ] Follows the code style of this project.
- [ ] Tests Cover Changes
- [ ] Documentation

### CODE REVIEW

**Comment by H34D** [comment]
please fix red builds before merging

**Comment by aabdulwahed** [comment]
@H34D It is still work in progress but I will do. The update is very simple but fixing the issues that are reported by the tool in the contracts might takes one~two days.

**Comment by H34D** [comment]
conflicts again @aabdulwahed

File: package.json [hunk 1]
```diff
@@ -55,5 +58,9 @@
     "truffle": "^5.0.0",
     "truffle-hdwallet-provider": "^1.0.1",
     "web3": "1.0.0-beta.37"
+  },
+  "dependencies": {
```
Comment (H34D) [inline package.json]: they should go devDependencies since we don't release code on NPM

Comment (jcortejoso) [reply package.json]: Moved 👍

**Pull request merged**